esmvaltool/utils/recipe_test_workflow/recipe_test_workflow/app/compare/bin/compare_task_runner.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+# Send the output from 'set -x' to 'stdout' rather than 'stderr'.
+BASH_XTRACEFD=1
+set -eux
+
+FOLDER_NAME="${OUTPUT_DIR}/recipe_${KGO_METRIC}*"
+KGO="${KGO_ROOT_PATH}/recipe_${KGO_METRIC}*"
+
+# Create a variable that defines the path to the compare script.
+COMPARE_SCRIPT=${ESMVALTOOL_DIR}/esmvaltool/utils/testing/regression/compare.py
```

Comment:
This should have quotes, apologies for missing them when I provided you with this line!
```suggestion
COMPARE_SCRIPT="${ESMVALTOOL_DIR}/esmvaltool/utils/testing/regression/compare.py"
```